PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceConfigFormPanelAdvanced.java

```diff
@@ -142,6 +145,10 @@ public void setDeploymentVisible(boolean visible) {
         this.selectorApplication.setVisible(visible);
     }
 
+    public SubscriptionComboBox getSelectorSubscription() {
+        return selectorSubscription;
+    }
```

Comment:
it's NG to make it public.
I would prefer making `selectorSubscription` `protected` and then override the `init` method to add another listener.